src/vsr/replica.zig

```diff
@@ -3767,23 +3885,15 @@ pub fn ReplicaType(
             return op;
         }
 
-        /// Finds the header with the highest op number in a slice of headers from a replica.
-        /// The headers must be continuous, in reverse order, all connected, and with no gaps.
-        fn op_highest(headers: []const Header) u64 {
```

Comment:
Formerly just used for an assertion, but we already have an even stronger version of this via `message_body_as_headers_chain_disjoint`.